### DISCUSSION TITLE
[求助]如何支走派派拐走露西呢？在线等，急！！！

### DISCUSSION THREAD

**Discussion by BITE-7274** [share121/inter-knot#1063] [其他]
![20240714174236](https://github.com/user-attachments/assets/9aa9eebb-168f-4029-905c-b222939ab9f7)

**Comment by axuan0** [comment]
让我拐走派派就行了

**Comment by Qiling-XwX** [comment]
等她们走到没人地方，把迷药抹到布上，直接捂住口鼻，然后装进麻袋即可，这样两个都能拐走了

**Comment by BITE-7274** [comment]
@axuan0 我已经花了3000丁尼请她们吃拉面了，你人呢！快！！！

**Comment by Qiling-XwX** [comment]
![B6562E79DAA5B0BF62EB56853953A398](https://github.com/user-attachments/assets/68d7ead5-cf28-4c4c-b2ce-7bf58a7076a0)

**Comment by DipleMoss** [comment]
会被治安局抓去喝茶的，还是算了吧...

**Comment by Pessimi-AFK** [comment]
这个委托我接了！

**Comment by sdsd122** [comment]
@治安局　这人有点问题　麻烦警官注意一下他

**Comment by lzxixizi** [comment]
派派，我真的喜欢你啊😭😭😭
跟我走吧派派，为了你，我什么都会做的😭😭😭
求求你了派派，跟我走吧😭😭😭

**Reply by dont-continue** [reply]
@治安局，过来处理一下😎

**Comment by SHENGWANG10086** [comment]
蹲一个后续

**Reply by BITE-7274** [reply]
后续有了·-·

**Comment by ZhuYuan-148** [comment]
对未成年下手的后果…你应该懂吧
![Screenshot_20240810_173930](https://github.com/user-attachments/assets/ecb84302-f2f5-4f8c-9866-ad929d9ecff6)

**Comment by Lanxey** [comment]
有一说一，现有的角色当中最戳我的就是露西了[hide]每次看到她都想舔大腿[/hide]

**Comment by BITE-7274** [comment]
治安局已经到我拉面馆了，一楼你人呢！！！我都请她们吃10碗了，等我出狱后这30000丁尼定要你加倍奉...哎！哎！让我说完啊！！！
派派：露西，这次出来的费用应该够我买彩票了吧·-·
露西：想...想都别想，我...我也有要买的............只能买一个哦QwQ

**Comment by Qiling-XwX** [comment]
被制裁了（）

**Comment by FFT66** [comment]
绳网非法外之地！
